README.md

```diff
@@ -7,44 +7,45 @@ Build a stand-alone test harness with dynamic data to quickly prototype and isol
 [![MIT Licensed](http://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](http://opensource.org/licenses/MIT)
 
 ## Installation steps to setup and configure
-1. `npm install tuxharness --save-dev`
-1. Define a **recipe** file -- similar to *Gruntfile.js* or *gulpfile.js*
-	* Mac/Unix copy file `cp -nv "node_modules/tuxharness/tuxfile.js" "tuxfile.js"`
-	* Windows copy file `copy "node_modules/tuxharness/tuxfile.js" "tuxfile.js"`
-	* More example recipes [starter sample tuxharness recipe](test/starter.js) or [full sample](test/sample.js)
-1. Run the harness server
-	* `node ./node_modules/tuxharness/index.js`
-	* or add a *script* reference to your package.json like `"harness": "node ./node_modules/tuxharness/index.js"` then execute with `npm run harness`
+1. `npm init` in your project folder
+2. `npm install tuxharness --save-dev`
+3. Define a **recipe** file -- similar to *Gruntfile.js* or *gulpfile.js*
+    * Mac/Unix copy file `cp -nv "node_modules/tuxharness/tuxfile.js" "tuxfile.js"`
+    * Windows copy file `copy "node_modules/tuxharness/tuxfile.js" "tuxfile.js"`
+    * More example recipes [starter sample tuxharness recipe](test/starter.js) or [full sample](test/sample.js)
+4. Run the harness server
+    * `node ./node_modules/tuxharness/index.js`
+    * or add a *script* reference to your package.json like `"harness": "node ./node_modules/tuxharness/index.js"` then execute with `npm run harness`
```

Comment:
Note, in markdown syntax 1dot repeated is the same as ol li elements with the li repeating.
[Source](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)